Fix #892: Fix start menu entry title on Windows

show981111 opened the pull request:
#### Related issue
Issue #892 

#### Context / Background
Start menu entry for Windows is time-to-leave, not "Time to Leave"

#### What change is being introduced by this PR?
1) When using an electron package manager, I changed the app name from "time-to-leave" to "Time to Leave". This changed the application name inside the release-build from "time-to-leave" to "Time to Leave."


#### How will this be tested?
1) Run `npm run package:win`, and go to the release-builds folder. You will see "Time to Leave" application not "time-to-leave"
2) Then run `npm run setup:win` in `packages/` there will be a setup.exe file. If you install Time to Leave with this file, you will get `Time to Leave` as a start menu entry

araujoarthur0 commented on css/styles.css:
```diff
@@ -314,6 +314,7 @@ body {
 
 .waiver-img {
   filter: invert(var(--svg-invert));
+  display: none;
```
Could you please open another PR with this change? I know it's coming from the npm automatic runs, but it's not related to this PR.

show981111 replied:
Just did  #902

araujoarthur0 reviewed (changes requested):
Can you please add an image showing it on the windows menu after your change?

show981111 commented:
> Can you please add an image showing it on the windows menu after your change?

I am not sure how to build setup.exe but this is what you see after you build a package for windows.
![IMG_3472](https://user-images.githubusercontent.com/57741570/201499027-82af8727-f84c-4ce7-b721-74b0b8c8d66e.jpg)

araujoarthur0 commented:
> > Can you please add an image showing it on the windows menu after your change?
> 
> I am not sure how to build setup.exe but this is what you see after you build a package for windows. ![IMG_3472](https://user-images.githubusercontent.com/57741570/201499027-82af8727-f84c-4ce7-b721-74b0b8c8d66e.jpg)

I believe there is an npm rule that does the packaging. Can you check the rules in package.json?

show981111 commented:
> > > Can you please add an image showing it on the windows menu after your change?
> > 
> > 
> > I am not sure how to build setup.exe but this is what you see after you build a package for windows. ![IMG_3472](https://user-images.githubusercontent.com/57741570/201499027-82af8727-f84c-4ce7-b721-74b0b8c8d66e.jpg)
> 
> I believe there is an npm rule that does the packaging. Can you check the rules in package.json?

I checked the package.json script and looks like `npm run package:win` is the correct one. 
Also, the releasing documentation said `npm run package:win` will create an executable application, and the application you get after running this command is in the photo I posted before.

tupaschoal reviewed (commented):
The exe is just one part. The original issue mentions the name of the app when installed on Windows itself.

I can't test if this patch fixes it or not, because the release steps on it do not work after this change, since it changes the folder from `release-builds\time-to-leave-win32-ia32\` to `release-builds\Time to Leave-win32-ia32\` and `npx electron-installer-windows` stops working

show981111 commented:
> The exe is just one part. The original issue mentions the name of the app when installed on Windows itself.
> 
> I can't test if this patch fixes it or not, because the release steps on it do not work after this change, since it changes the folder from `release-builds\time-to-leave-win32-ia32\` to `release-builds\Time to Leave-win32-ia32\` and `npx electron-installer-windows` stops working

I was able to fix it by adding the arguments on `npx electron-installer-windows`  
I am not sure what arguments we are using currently, but it seems like the name of the start menu follows the name of the folder. I tried various options in electron-installer-windows but it did not work, but it worked after I changed the folder name.

I added the working command line in package.json. 
You can test this by running `npm run setup`
Below is the image of start menu.
![time_to_leave](https://user-images.githubusercontent.com/57741570/202568385-11e8d428-8271-4473-b3c8-a6ced086b9c3.png)

tupaschoal commented on package.json:
```diff
@@ -26,14 +26,15 @@
     "test:jest": "jest --runInBand --detectOpenHandles --verbose --colors",
-    "test:mocha": "nyc --reporter=lcov --reporter=json mocha tests/*"
+    "test:mocha": "nyc --reporter=lcov --reporter=json mocha tests/*",
+    "setup" : "npx electron-installer-windows --src ./release-builds/\"Time to Leave-win32-ia32\"/ --dest ./release-builds/\"Time to Leave-win32-ia32\"/installers/ --exe \"Time to Leave.exe\""
```
```suggestion
    "setup:win" : "npx electron-installer-windows --src ./release-builds/\"Time to Leave-win32-ia32\"/ --dest 'packages\' --verbose --exe \"Time to Leave.exe\""
```
I suggest renaming this to `setup:win` and reusing it on `.github\workflows\Release.yml`. I brought the `dest` configuration from there.

Also, any reason for naming the exe explicitly? Would it be "time-to-leave" if not?

show981111 replied:
If we use "Time to Leave" for application name, then it will create Time to Leave.exe. Default is, according to .nuspec, time-to-leave.exe. Thus, if I do not specify, it said it failed on finding corresponding .exe file. 
Also, is there a way to test release? I want to make sure my release process  works correctly.

araujoarthur0 replied:
I'm not sure there's a way to test the release without actually releasing it here. I haven't played with the actions yet.
Maybe you can run the actions in your fork?

show981111 replied:
Thank you for the suggestion. I was able to run the action in my fork and it passed.

araujoarthur0 replied:
Great!

tupaschoal reviewed (commented):
It works now, thanks, but see my comments below

araujoarthur0 commented:
Can you please update the title now that you separated the css change?

tupaschoal commented:
\changelog-update
Message: Fix [#892]: start menu entry title on Windows is hyphenated
